docs/index.rst

```diff
@@ -103,6 +103,18 @@ The ``openapi`` directive supports the following options:
 ``exclude``, ``include`` and ``paths`` can also be used together (``exclude``
 taking precedence over ``include`` and ``paths``)
 
+``http-methods-order``
+  A whitespace delimited list of HTTP methods to render first. For example:
+
+  .. code:: restructuredtext
+
+     .. openapi:: specs/openapi.yml
+        :http-methods-order:
+            head
+            get
+
+  Would render the ``head`` method, followed by the ``get`` method, followed by the rest of the methods in their declared ordered.
+
```

Comment:
At this moment, documentation is about old renderer, and is not about new one. I was thinking about releasing v1.0 with the new renderer by default once it's feature complete. So we can probably keep it.